### DISCUSSION TITLE
Waypoints and customization there of

### DISCUSSION THREAD

**Discussion by LazerSharkGaming** [EssentialsX/Essentials#4015] [Ideas and Feature Suggestions] [answered]
Could you add the commands:
/wp                          <name>
/waypoint               <name> 
/wpAdd                   <name> (optional: <(colors)/type>)
/waypointAdd        <name> (optional: <(colors)/type>)
/wpRemove            <name>
/waypointRemove <name>
/wpd                         <name>  <(colors)/type/on/off/particle>
/waypointDisplay  <name>  <(colors)/type/on/off/particle>

the main differences from warps being that waypoints have 2 things at location and they are per person
items at location:
customizable floating tag
beam/beacon/particle beacon/etc.

**Comment by JRoy** [comment]
Thanks for the feature request but I'm afraid this is outside the scope of EssentialX.

**Comment by mdcfe** [comment, marked as answer]
To add to @JRoy's comment, warps aren't really intended to be a visual feature. You could just build a beacon and create a hologram using HolographicDisplays, or for something more automated, an external plugin could retrieve warp locations through EssentialsX's API and apply effects at those locations.
